### ISSUE TITLE
35756905: Both macOS and Safari should allow tabbing to all controls by default

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#18986]
#### Description

A keyboard-only user who is trying Mac for the first time, or someone testing accessibility on a Mac, or a keyboard power-user switching from Windows to Mac - all of these people are immediately stumped and surprised that the Tab key does not traverse to all focusable controls.
They may google it, or ask someone, or stumble through the UI, (or give up!), but eventually they usually figure out that there is a system setting to allow tabbing to all controls in macOS desktop apps... and even later still, they find out that there's yet another setting change required to be able to traverse focusable controls with the keyboard in Safari.

These 2 settings are both described in this article: http://www.weba11y.com/blog/2014/07/07/keyboard-navigation-in-mac-browsers/

Here's a discussion about this problem on an accessibility Slack channel:
https://web-a11y.slack.com/archives/C042TSFGN/p1508862816000611

Please seriously consider making both of these settings default to allowing tabbing to all controls in both macOS and in Safari.

-
Product Version: All macOS since the beginning
Created: 2017-11-29T22:11:21.958260
Originated: 2017-11-29T17:09:00
Open Radar Link: http://www.openradar.me/35756905
